changelog.d/5694.misc

```diff
@@ -0,0 +1 @@
+Makeaeger fully configurable Jaeger fully configurable.
```

Comment:
What now?

Reply:
I'm going to disable middle click paste..